react/tic-tac-toe/src/utils/Validations/index.js

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line no-useless-escape
+const VALID_EMAIL = /^\w+([\.-]?\w+)*@wolox.com/;
+
+const isRequired = value => value ? undefined : 'Required field';
```

Comment:
Returning undefined in JS is a bad practice. Since what `undefined` means is that no value has been set in a variable, not even once. I would use null instead when possible